### PR TITLE
fix: handle image previews for unpublished uploads (fix #472)

### LINKED ISSUE #472: Images uploaded from the local drive are not previewed in Netlify CMS
## Describe the bug

When using Netlify CMS to edit content, if an image field uses an image uploaded from the local drive, the preview pane doesn't display the uploaded image.

However, if the image is sourced from an image URL, the preview pane display the image properly.

## To Reproduce

1. Go to a deploy preview, or a local Netlify CMS by running "npm run start";
2. Select an initiatives record from the initiatives collection;
3. In the cover image field, select "choose different image";
4. In the popup dialog, select the "upload" button, select an image from the local drive, click "choose selected";
5. The popup dialog closes. On the preview pane of the editing page, the old image has been removed but the new uploaded image is not displayed.

## Expected behavior

The preview pane displays the new uploaded image.

### PR DESCRIPTION
**Pull request by greatislander**
* [x] This pull request has been linted by running `npm run lint` without errors
* [x] This pull request has been tested by running `npm run start` and reviewing affected routes
* [x] This pull request has been built by running `npm run build` without errors
* [x] This isn't a duplicate of an existing pull request

## Description

Ensures that uploaded images are always previewed properly.

## Steps to test

1. Run `npm start`.
2. From the CMS, create a new initiative and upload a cover image.

**Expected behavior:** The image should appear in the preview.

## Additional information

Uses `getAsset()`: https://www.netlifycms.org/docs/customization/#registerpreviewtemplate

## Related issues

Resolves #472.

### CODE REVIEW

File: src/admin/cms.js [hunk 1]
```diff
@@ -32,27 +32,29 @@ Preview.propTypes = {
 CMS.registerWidget("uuid", UuidControl, UuidPreview);
 CMS.registerPreviewStyle("/css/main.css");
 
-const Initative = ({ entry }) => (
-	<Preview
+const Initative = ({ entry, getAsset }) => {
+	console.log();
```
Comment (cindyli) [inline src/admin/cms.js:36]: This stray can be removed.

Comment (greatislander) [reply src/admin/cms.js:36]: Oops! Thanks @cindyli.

**Comment by cindyli** [comment]
We'll be ready to promote the dev to the main soon. I'm thinking it would be nice have a section in the README about how content editors use Netlify CMS. If you plan to have the wiki page you are working on as a generic entry for all Netlify CMS related information, this section can simply contain a paragraph pointing to the proper section in the wiki page. Otherwise, we probably should describe CMS related UI and steps in this section. What do you think, @greatislander?

If you agree, shall we take the chance of this last PR before the merge to update the README?

**Comment by greatislander** [comment]
@cindyli I'm not sure the README is the place for it, as I don't expect that many of the content editors are using GitHub. I agree some guidelines are needed but I think they should be on the Wiki, and we should email the We Count list with a link to the appropriate wiki page once they're ready. I can work on them today. I think we should go ahead and merge this and promote the changes in dev to main.

**Comment by cindyli** [comment]
OK. Let's move on with your suggestion.

**Review by cindyli — APPROVED** [review]

**Pull request merged**